fachhochschule-vorarlberg-author-date.csl

```diff
@@ -92,54 +96,39 @@
   </macro>
   <macro name="editor">
     <choose>
-      <if variable="author">
-        <choose>
-          <if type="chapter">
-            <names variable="editor" delimiter="; ">
-              <name name-as-sort-order="all" sort-separator=", " delimiter="; " delimiter-precedes-last="never" form="long" suffix=""/>
-              <label form="short" prefix=" (" suffix=".): "/>
-            </names>
-          </if>
-        </choose>
-      </if>
+      <if variable="author"></if>
```

Comment:
@cookie365, I think you can remove the entire `<choose/>` here to test for the presence of the "author" variable. If the "editor" variable has already been used through `<substitute/>` in the "author" macro, it should be automatically suppressed in the rest of the style. (see http://docs.citationstyles.org/en/stable/specification.html#substitute for background)